analyzers/src/SonarAnalyzer.CSharp/Rules/InfiniteRecursion.RoslynCfg.cs

```diff
@@ -99,6 +97,18 @@ operation.Kind switch
                             OperationKindEx.Invocation
                                 when IInvocationOperationWrapper.FromOperation(operation) is var invocation && (!invocation.IsVirtual || InstanceReferencesThis(invocation.Instance)) =>
                                 invocation.TargetMethod,
+                            OperationKindEx.Binary
+                                when IBinaryOperationWrapper.FromOperation(operation) is var binaryOperation =>
+                                binaryOperation.OperatorMethod,
+                            OperationKindEx.Decrement
+                                when IIncrementOrDecrementOperationWrapper.FromOperation(operation) is var decrementOperation =>
+                                decrementOperation.OperatorMethod,
+                            OperationKindEx.Increment
+                                when IIncrementOrDecrementOperationWrapper.FromOperation(operation) is var incrementOperation =>
+                                incrementOperation.OperatorMethod,
+                            OperationKindEx.Unary
+                                when IUnaryOperationWrapper.FromOperation(operation) is var bitwiseOperation =>
```

Comment:
this is not necessarily a bitwise operator, it can also be 

- Unary plus and minus operators
- Logical negation operator !
- Bitwise complement operator ~

please rename